client/lib/invites/actions.js

```diff
@@ -51,7 +51,9 @@ export function fetchInvite( siteId, inviteKey ) {
 		} );
 
 		if ( error ) {
-			analytics.tracks.recordEvent( 'calypso_invite_validation_failure' );
+			analytics.tracks.recordEvent( 'calypso_invite_validation_failure', {
+				error: error.error
```

Comment:
In each case, since we are already checking the existence of `error` before logging an event, the worst case should be that `error.error` is `undefined`.